src/parser/test/ParserTest.cpp

```diff
@@ -474,6 +474,12 @@ TEST(Parser, ColumnSpacesTest) {
 }
 
 TEST(Parser, IndexOperation) {
+    {
+        GQLParser parser;
+        std::string query = "CREATE TAG INDEX empty_field_index ON person()";
```

Comment:
What's the meaning for empty field index?

Reply:
refer to https://github.com/vesoft-inc/nebula-storage/pull/174

use to get all vertices on tag or edges on one edgetype

Reply:
Emm.Mix the sentence to create properties index and tag/edge index is a little ambiguous

Reply:
I think it's ok.
You think it just creating an index of 0 fields to get the vertices of the tag or the edges on the edgeType.

Similar to the logic of a normal index, when an index with one or more fields is included, it is not allowed to create an index with zero fields.